app/models/manageiq/providers/lenovo/physical_infra_manager/refresher.rb

```diff
@@ -1,10 +1,15 @@
 module ManageIQ::Providers::Lenovo
   class PhysicalInfraManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
     def parse_legacy_inventory(ems)
-      log_header = "MIQ_LENOVO(#{self.class.name}.#{__method__} Calling for [#{ems.name}])"
-      $log.info("#{log_header}")
+      $log.info("MIQ_LENOVO(#{self.class.name}.#{__method__} Calling for [#{ems.name}])")
 
-      ManageIQ::Providers::Lenovo::PhysicalInfraManager::RefreshParser.ems_inv_to_hashes(ems, refresher_options)
+      # Perform a full fetch of the EMS inventory
+      log_header = "Collecting data for EMS : [#{ems.name}] id: [#{ems.id} ref: #{ems.uid_ems}]"
```

Comment:
Oh I see you moved them here, we have them in ems_inv_to_hashes because some providers collect and parse in two different steps.  This helps isolate issues with connectivity to the provider, I would recommend leaving them there so that we could do that in the future.